cypress/support/commands.js

```diff
@@ -32,10 +32,33 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
+Cypress.Commands.add('register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users', {
+      email,
+      username,
+      password
+    });
+  });
+
+Cypress.Commands.add('login',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
```

Comment:
The `username` parameter is not typically required for login operations. Consider removing it from the `login` command unless your backend specifically requires it.